paasta_tools/kubernetes_tools.py

```diff
@@ -2707,6 +2707,12 @@ def get_all_pods(kube_client: KubeClient, namespace: str = "paasta") -> Sequence
     return kube_client.core.list_namespaced_pod(namespace=namespace).items
 
 
+def get_pods_in_all_namespaces(
+    kube_client: KubeClient, label_selector: str = "paasta.yelp.com/service"
+) -> Sequence[V1Pod]:
```

Comment:
normally I'd write some words here about how we should probably prefer more concrete types over ABCs, but we might as well be consistent with the rest of this file :p